agent/taskresource/envFiles/envfile.go

```diff
@@ -358,9 +359,32 @@ func (envfile *EnvironmentFileResource) writeEnvFile(writeFunc func(file oswrapp
 	return nil
 }
 
-// Cleanup removes env files downloaded for the task
+// Cleanup removes all env files downloaded for the task
```

Comment:
why not just remove the whole `/var/lib/ecs/data/envfiles/cluster_name/task_id/` directory? i don't think we should leave an empty `/var/lib/ecs/data/envfiles/cluster_name/task_id/` folder behind when the task is gone

Reply:
Good point. I'll change it.